source/common/http/websocket/ws_handler_impl.cc

```diff
@@ -46,6 +48,11 @@ Network::FilterStatus WsHandlerImpl::onData(Buffer::Instance& data, bool end_str
   ENVOY_LOG(debug, "WsHandlerImpl::onData with buffer length {}, end_stream == {}", data.length(),
             end_stream);
 
+  if (init_failure_) {
+    ENVOY_LOG(debug, "WsHandlerImpl::onData init_failure_ == true; discarding");
```

Comment:
should be trace?

Reply:
All other logging in this function is at level debug, so I'll leave it this way for consistency.

Reply:
And then I went and touched all of it, so I'll change it to trace.